ajax/libs/react-mdl/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "react-mdl",
+  "filename": "ReactMDL.min.js",
+  "version": "1.10.3",
+  "description": "React Components for Material Design Lite",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/react-mdl/react-mdl"
```

Comment:
The link always redirect to https://github.com/tleunen/react-mdl.
I know you just fixed it.
But there is no user or organization named `react-mdl`.
Maybe revise it again?
Many thanks.